### PR TITLE
Tests: M2-3897 - useHeaderSticky, useLogout

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

**Review by IhorNesterenko99 — APPROVED** [review]

File: src/shared/hooks/useLogout.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    mockAxios.post.mockImplementation((url: string) => {
+      expect(url).toBe('auth/logout2');
+    });
+
+    await waitFor(() => {
+      result.current();
+    });
+
+    expect(mockedDispatch).toBeCalledTimes(3);
+    expect(mockedNavigate).toBeCalledWith(page.login);
```
Comment (wildkiomi) [inline src/shared/hooks/useLogout.test.ts:64]: could use nthCalledWith

File: src/shared/hooks/useLogout.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    await waitFor(() => {
+      result.current();
+    });
+
+    expect(mockedNavigate).toBeCalledWith(page.login);
+  });
+
+  test('deleting access token resets all data', async () => {
+    const { result } = renderHook(() => useLogout());
```
Comment (wildkiomi) [inline src/shared/hooks/useLogout.test.ts:37]: renderHook(useLogout)

**Review by alex-i-e — APPROVED** [review]

File: src/shared/hooks/useLogout.test.ts (hunk 1 above)
Comment (muhurov) [reply src/shared/hooks/useLogout.test.ts:81]: done

File: src/shared/hooks/useLogout.test.ts (hunk 2 above)
Comment (muhurov) [reply src/shared/hooks/useLogout.test.ts]: done

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**